ruma-events-macros/src/event.rs

```diff
@@ -37,7 +43,12 @@ pub fn expand_event(input: DeriveInput) -> syn::Result<TokenStream> {
         .iter()
         .map(|field| {
             let name = field.ident.as_ref().unwrap();
-            if name == "content" && ident.to_string().contains("Redacted") {
+            if name == "content" && matches!(
+                var,
+                EventKindVariation::Redacted
+                    | EventKindVariation::RedactedSync
+                    | EventKindVariation::RedactedStripped
```

Comment:
Do we have checks like this in many places? Could move it into a method on `EventKindVariation`.